src/cljc/athens/components_util.cljc

```diff
@@ -0,0 +1,22 @@
+(ns athens.components-util
+  (:require
+    [athens.components.todo :as todo]
+    [athens.components.website-embeds :as website-embeds]))
+
+
+(defn default-component
+  [content uid]
+  [:span {:id uid} [:button {:class "component-default"} content]])
```

Comment:
Don't use `id` and `class` here

Reply:
Regarding `id` and `class` here (and [here](https://github.com/athensresearch/athens/pull/274#discussion_r456835720) and [here](https://github.com/athensresearch/athens/pull/274#discussion_r456835405)), I'd say that it's more for the sake for future themability (like how Roam can make the buttons/component elements look different based on different `roam/css`). If we're doing this though, we should document them for future theme-makers to reference. Or we can do it at some later time.

Reply:
Have had a few early discussions with @shanberg about custom theming. Main point: we think we can do better than `roam/css`. Main reason I don't want to not-yet-used classes/ids yet is because our current classes/ids are kind of a mess and will require a refactor soon.

Reply:
Makes sense! I'll just hold off adding them so we can talk later